### DISCUSSION TITLE
Add option to play sound when completing a habit

### DISCUSSION THREAD

**Discussion by pech2** [iSoron/uhabits#654] [Feature Requests]
Feature request to add option to vibrate and/or play sound when completing a habit.

**Comment by iSoron** [comment]
Thank you for the feature request, @pccchiang. Playing a sound after a habit is completed would be nice. I'm not sure whether this should be enabled by default, but at least it should be an option. Pull requests are welcome.

Regarding vibration, if you have "touch vibration" enabled in your phone settings (Settings / Sound) and your phone is not mute, then the app should already vibrate when adding a checkmark. If it doesn't, that's a bug.

**Reply by patrik-bartak** [reply]
If such a sound feature was implemented as an optional preference, can it also be implemented for the checkmark widget? I noticed the checkmark widget does not use the vibration feature you mention here, and I am not sure why.

**Comment by auipga** [comment]
An applause like in freeletics sports app after a completed workout or a text-to-speech "you did it!" would be nice indeed. +1
